packages/app-extensions/src/rest/helpers.spec.js

```diff
@@ -159,7 +159,19 @@ describe('app-extensions', () => {
 
       describe('fetchDisplay', () => {
         test('should call fetch', () => {
-          const response = {display: 'Test'}
+          const response = {
```

Comment:
it would be nice to have a test that verifies that only non-cached displays are loaded and that verifies, that cached and loaded displays are combined in the answer